src/test/java/HttpServerTest.java

```diff
@@ -0,0 +1,104 @@
+import org.apache.http.HttpResponse;
+import org.apache.http.client.fluent.Request;
+import org.junit.*;
+import org.mockito.Mockito;
+import org.rapidoid.net.Server;
+import ru.kontur.airlock.EventSender;
+import ru.kontur.airlock.HttpServer;
+import ru.kontur.airlock.ValidatorFactory;
+import ru.kontur.airlock.dto.AirlockMessage;
+import ru.kontur.airlock.dto.EventGroup;
+import org.apache.http.HttpStatus;
+import java.util.HashMap;
+import java.util.Properties;
+
+import static org.mockito.Mockito.*;
+
+public class HttpServerTest {
+    static Server server;
+    private static final int port = 12121;
+
+    @BeforeClass
+    public static void init() throws Exception {
+        final Properties properties = new Properties();
+        properties.setProperty("log4j.rootLogger","DEBUG,CONSOLE");
+        properties.setProperty("log4j.appender.CONSOLE","org.apache.log4j.ConsoleAppender");
+        properties.setProperty("log4j.appender.CONSOLE.threshold","debug");
+        org.apache.log4j.PropertyConfigurator.configure(properties);
+        org.apache.log4j.BasicConfigurator.configure();
+
+        final EventSender eventSenderMock = mock(EventSender.class);
+        doNothing().when(eventSenderMock).send(Mockito.any(EventGroup.class));
+        final HashMap<String, String[]> hashMap = new HashMap<>();
```

Comment:
It's preferred to use variable of type Map (which is interface) instead of HashMap (which is implementation).